.github/workflows/fuzzydata-test.yml

```diff
@@ -0,0 +1,33 @@
+name: fuzzy
+on:
+  pull_request:
+jobs:
+  test-fuzzydata:
```

Comment:
We should probably add a strategy for testing of Modin both on Ray and Dask engines as it is done here.
https://github.com/modin-project/modin/blob/d5ff94ea75b73e8b928bda1ec6b0363d5125207a/.github/workflows/ci.yml#L422

Reply:
Sounds like a good idea. I should try to find a good way to pass on the test configuration to fuzzydata so that the results can be annotated.